scripts/gen_ref_nav.py

```diff
@@ -42,7 +42,8 @@
     elif parts[-1].startswith("_"):
         continue
 
-    nav[tuple(parts)] = doc_path.as_posix()
+    nav_parts = [f"<code>{part}</code>" for part in parts]
```

Comment:
Wrapping the navigation items in code spans prevents check spelling errors, but `<code>...</code>` then appears in the browser tab title. The alternative is to add the module names to the dictionary of known words. Let me know what you'd prefer.

Reply:
Can you show me an example?

Reply:
The index looks like this:
![markdown_apidocs_sitemap](https://github.com/Python-Markdown/markdown/assets/3999221/820dbe91-4408-441e-9cb8-717a6bcd065b)

And the tab title looks like this:
![markdown_apidoc_tab](https://github.com/Python-Markdown/markdown/assets/3999221/c5ad0432-a45f-4861-988c-fa0a009aaf4e)

Absolute top is my window manager bar, then below are the Firefox tabs. Also note the breadcrumbs being styled as code.

Reply:
Thanks for that. I thought that was what you were saying. I imagine that might be something to address in the theme. I'll take a look at it. Leave it as-is for now.

Reply:
It's easy to remove HTML tags from the title, within the theme, thanks to Jinja's [striptags filter](https://jinja.palletsprojects.com/en/3.0.x/templates/#jinja-filters.striptags):

```jinja+html
<title>{{ page.title | striptags }}</title>
```

If that's something you'd like to consider.

Reply:
I fixed this with the changes below, which overrides the  `htmltitle` block of the theme's base template and runs the page title through the `striptags` filter. However, I would rather fix this upstream in the theme. I'll report back if/when that happens.

```diff
diff --git a/custom_theme/main.html b/custom_theme/main.html
new file mode 100644
index 0000000..9f9cdd5
--- /dev/null
+++ b/custom_theme/main.html
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{#
+Override the title block of the nature Theme.
+#}
+
+      {%- block htmltitle %}
+        <title>{% if page and page.title %}{{ page.title|striptags }} &#8212; {% endif %}{{ config.site_name }} {{ config.theme.release }} documentation</title>
+      {%- endblock %}
diff --git a/mkdocs.yml b/mkdocs.yml
index 43fe090..c2502b3 100644
--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -12,6 +12,7 @@ theme:
   icon: py.png
   release: !!python/name:markdown.__version__
   issue_tracker: https://github.com/Python-Markdown/markdown/issues
+  custom_dir: custom_theme/

 extra_css:
   - custom.css
```

Reply:
I expect the changes in waylan/mkdocs-nature#5 to be available in the next release of the theme. So we can specify the dependency `mkdocs-nature>=0.6` to address this.

Reply:
I've upgraded the version in pyproject.toml, CI will likely fail until 0.6 is actually released :slightly_smiling_face: